angular-cli-build.js

```diff
@@ -5,16 +5,53 @@ const path = require('path');
 // Import the require hook. Enables us to require TS files natively.
 require('ts-node/register');
 
-
 const Angular2App = require('angular-cli/lib/broccoli/angular2-app');
 const Funnel = require('broccoli-funnel');
 const MergeTree = require('broccoli-merge-trees');
 const autoPrefixerTree = require('broccoli-autoprefixer');
 
 
 module.exports = function(defaults) {
-  var angularAppTree = new Angular2App(defaults, {
-    sourceDir: 'src/',
+  const appTree = buildAppTree(defaults);
+  const cssAutoprefixed = autoPrefixerTree(new Funnel(appTree, {
```

Comment:
The changes in this file would use some high-level comments explaining what's going on.